src/connectors/aws_collect.py

```diff
@@ -1008,6 +1069,10 @@ def process_aws_response(task, page):
             ):
                 yield x
 
+    except ParamValidationError as e:
+        # hack(aku): Should do something about e
+        pass
+
```

Comment:
@andrey-snowflake This error came up while I was trying to insert list-findings results into describe-findings (sometimes I would get an empty list). This exception doesn't have a `response` attribute.